sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/WritePartition.java

```diff
@@ -126,8 +129,8 @@ public void processElement(ProcessContext c) throws Exception {
       // Return a null destination in this case - the constant DynamicDestinations class will
```

Comment:
This comment is out of date -- it doesn't return a null destination, it returns the default destination according to dynamiDestinations.getDestination(...)